core/shared/src/main/scala/org/http4s/headers/Sec-WebSocket-Accept.scala

```diff
@@ -27,7 +28,7 @@ import scodec.bits.ByteVector
 import java.util.Base64
 import scala.util.Try
 
-final class `Sec-WebSocket-Accept`(hashBytes: ByteVector) {
+final case class `Sec-WebSocket-Accept`(hashBytes: ByteVector) {
```

Comment:
Do we actually want `hashBytes` to be accessible?

Reply:
Ah I didn't think of this. I changed it to case class for the `equals` so that it passes the `HeaderLaws` test. Is there any reason why it should not be accessible?

Reply:
I think this is fine. Most/all of our headers are `case class` with public members.

We should confirm that `hashBytes` is the best name for it. In MDN they call it `hashedKey` maybe?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-WebSocket-Accept